modules/ign.js

```diff
@@ -0,0 +1,220 @@
+  const PAGESIZE = 60; // Entries per message for this module.
+  let system = interaction.options.getString("system").toLowerCase();
+  let username = interaction.options.getString('ign');
+  const foundSystem = findSystem(system);
+  if (!foundSystem) return interaction.reply({ content: `\`${system}\` isn't a valid system.`, ephemeral: true });
+  system = foundSystem?.system;
+  if (system == 'birthday') {
+    const bd = new Date(username);
+    if (bd == 'Invalid Date') return interaction.reply({ content: "I couldn't understand that date. Please use Month Day format (e.g. April 1 or 4/1)", ephemeral: true });
+    const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sept", "Oct", "Nov", "Dec"];
+    username = `${months[bd.getMonth()]} ${bd.getDate()}`;
+  }
+  const users = await Module.db.ign.findUsername(username, system);
+  if (users.length == 0) return interaction.reply({ content: `No members have the IGN ${username} for ${foundSystem.name}.`, ephemeral: true });
+  const guild = interaction.guild;
+  const embed = u.embed().setTitle(`The following members have the IGN ${username} for ${foundSystem.name}`);
+  const wePlay = users.map(user => guild.members.cache.get(user.discordId))
+    .filter(usr => usr != null)
+    .sort((a, b) => a.displayName.toLowerCase().localeCompare(b.displayName.toLowerCase()));
+
+  const listChunks = [];
+  for (let i = 0; i < wePlay.length; i += PAGESIZE) {
+    listChunks.push(wePlay.slice(i, i + PAGESIZE));
+  }
+  embed.setDescription(listChunks[0].join('\n'));
+  interaction.reply({ embeds: [embed] });
+  if (listChunks.length > 1) {
+    listChunks.slice(1).forEach(chunk => {
+      interaction.followUp({ content: chunk.join("\n") });
+    });
+  }
+}
+
+const Module = new Augur.Module()
+.addInteractionCommand({
+  name: "ign",
+  guildId: sf.ldsg,
+  commandId: sf.commands.slashIgn,
+  process: async (interaction) => {
+    switch (interaction.options.getSubcommand(true)) {
+    case "view": return slashIgnView(interaction);
+    case "set": return slashIgnSet(interaction);
+    case "remove": return slashIgnRemove(interaction);
+    case "whoplays": return slashIgnWhoplays(interaction);
+    case "whois": return slashIgnWhois(interaction);
+    }
+  }
+})
+.setInit(async () => {
+  const doc = new GoogleSpreadsheet(config.google.sheets.config);
+  try {
+    await doc.useServiceAccountAuth(config.google.creds);
+    await doc.loadInfo();
+    const sheet = await doc.sheetsByTitle["IGN"].getRows();
+    const aliasSheet = await doc.sheetsByTitle["IGN Aliases"].getRows();
+    aliases = new u.Collection(aliasSheet.map(x => [x["Alias"]?.toLowerCase(), x["System"]?.toLowerCase()])).concat(new u.Collection(sheet.map(x => [x["Name"]?.toLowerCase(), x["System"]?.toLowerCase()])));
+    IGNs = new u.Collection(sheet.map(x => [x["System"]?.toLowerCase(), { system: x["System"]?.toLowerCase(), name: x["Name"], category: x["Category"] ?? "Game Platforms", link: x["Link"] }]));
```

Comment:
```suggestion
    IGNs = new u.Collection(sheet.map(x => [x["System"]?.toLowerCase(), { system: x["System"]?.toLowerCase(), name: x["Name"], category: x["Category"] || "Game Platforms", link: x["Link"] }]));
```
If the field in the Google Sheet is empty, it shows up here as an empty string (not `null` or `undefined`). So we need to use `||` rather than `??` to fill in blanks.